AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -666,10 +666,13 @@ - (ASLayout *)transitionLayoutWithAnimation:(BOOL)animated
 
 - (ASLayout *)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize animated:(BOOL)animated
 {
+  BOOL disableImplicitHierarchyManagement = self.usesImplicitHierarchyManagement == NO;
```

Comment:
Either way is fine. This flag is going to be removed before 2.0 once I fix some of the ordering issues with the implicit hierarchy operations and also include support for manually managed nodes.